src/booleans.js

```diff
@@ -1,62 +1,65 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
-  // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !a && !b;
+}
 
 function one(a, b) {
-  // your code here
-};
+  return (a && !b) || (!a && b);
+}
 
 function truthiness(a) {
-  // your code here
-};
+  if (a) {
+    return true;
+  }
+  return false;
+}
```

Comment:
Can use a double bang operator `!` to convert to the truthiness of something. The first `!` would convert it to the opposite of it's truthiness, the second would flip this to the truthiness of the variable. i.e.
```
return !!a;
```